### PR TITLE
Fix header keyboard accessibility without removing checkbox

### PR DESCRIPTION
**Pull request by Yashs911**
Draft PR to solve some of the issues related to keyboard navigation.
Also fixes select dropdown focus 

### Screenshot

https://user-images.githubusercontent.com/64412143/114072269-b3580e00-98bf-11eb-989f-14d45009f15c.mp4

### Stakeholders

@bpmcneilly

### CODE REVIEW

**Comment by cdrini** [comment]
Can we drop the JS changes entirely for now? Would like to make sure @jdlrobson can keep his mind on his vacation :)

File: openlibrary/plugins/openlibrary/js/index.js [hunk 1]
```diff
@@ -187,6 +187,12 @@ jQuery(function () {
 
     $('#wikiselect').on('focus', function(){$(this).select();})
 
+    $(document).ready(function(){
+        $('.dropdown-checkbox').keydown(function(event) {
```
[the remark this answers is not in the document]
Comment (Yashs911) [reply openlibrary/plugins/openlibrary/js/index.js:191]: Should I start working on a new PR for the header (using the `<details>` tag method)?

[the remark this answers is not in the document]
Comment (jdlrobson) [reply openlibrary/plugins/openlibrary/js/index.js:191]: I recommend pushing ahead with #5087  yes. I'll leave a comment there.

**Comment by Yashs911** [comment]
Closing this PR as we will be implementing the `<details>` tag method.

**Pull request closed without merging**